src/NativeScript/Metadata/Metadata.mm

```diff
@@ -111,6 +118,63 @@ static int compareIdentifiers(const char* nullTerminated, const char* notNullTer
 
     return nullptr;
 }
+const MethodMeta* BaseClassMeta::member(const char* identifier, size_t length, MemberType type, size_t paramsCount, bool includeProtocols, bool onlyIfAvailable) const {
+    const std::vector<const MethodMeta*> metas = this->members(identifier, length, type, includeProtocols, onlyIfAvailable);
```

Comment:
`const MemberMeta* BaseClassMeta::member(const char* identifier, size_t length, MemberType type, bool includeProtocols, bool onlyIfAvailable) const {` can be implemented by calling `this->members` as well. And by doing so we will avoid a lot of duplicated code.
I know that if we call directly the current implementation it may have some performance hit, but let's measure the impact of the whole implementation and then decide what and where to optimize.